tests/test_config.py

```diff
@@ -199,6 +199,16 @@ def test_loading_keys_from_environment_variables(self):
         config.ready = True
         assert config.get("num_participants") == 1
 
+    def test_loading_auto_recruit_from_redis(self):
+        config = Configuration()
+        config.register("auto_recruit", bool)
+        config.set("auto_recruit", False)
+        from dallinger.db import redis_conn
+
+        redis_conn.set("auto_recruit", 1)
+        config.ready = True
```

Comment:
If you use the active_config fixture then you should be able to skip `config.ready`